src/Bank/Up.ts

```diff
@@ -0,0 +1,160 @@
+import {
+  BigDecimal,
+  Chunk,
+  Config,
+  DateTime,
+  Effect,
+  flow,
+  identity,
+  Layer,
+  Option,
+  pipe,
+  Redacted,
+  Schedule,
+  Stream,
+} from "effect"
+import { configProviderNested } from "../internal/utils.js"
+import {
+  HttpClient,
+  HttpClientRequest,
+  HttpClientResponse,
+} from "@effect/platform"
+import { NodeHttpClient } from "@effect/platform-node"
+import * as S from "@effect/schema/Schema"
+import { AccountTransaction, Bank } from "../Bank.js"
+
+const URL = "https://api.up.com.au/api/v1"
+
+export const UpBank = Effect.gen(function* () {
```

Comment:
```suggestion
export const UpLive = Effect.gen(function* () {
```

Just to be consistent with the Bnz bank.